sklearn/linear_model/_base.py

```diff
@@ -835,12 +790,6 @@ def _pre_fit(
 
     This function applies _preprocess_data and additionally computes the gram matrix
     `precompute` as needed as well as `Xy`.
-
-    Parameters
-    ----------
-    order : 'F', 'C' or None, default=None
-        Whether X and y will be forced to be fortran or c-style. Only relevant
-        if sample_weight is not None.
```

Comment:
Did you mean to remove this part of the doc string?

I was expecting `normalize` to be removed from the list of arguments of `_pre_fit` and in its code as well. Instead its still there and you updated the comment to say it will get removed in v1.4. I'm confused :D 

Is there something I can read to understand that? I think it would make it easier to help review this, otherwise I'll be wondering if a change is on purpose or not all the time

Reply:
> I was expecting normalize to be removed from the list of arguments of _pre_fit and in its code as well. Instead its still there and you updated the comment to say it will get removed in v1.4. I'm confused :D

When ``normalize`` has been deprecated, it was decided (don't ask me why :/ ) to remove it in 1.2 for estimators that had normalize=False by default and to change the default to False, to be then removed in 1.4, for estimators that had normalize=True by default.

Reply:
> Did you mean to remove this part of the doc string?

I did. it's not really related to this PR but when I was going through this function I noticed that this is completely irrelevant. There's no ``order`` parameter in this function.

Reply:
> When `normalize` has been deprecated, it was decided (don't ask me why :/ ) to remove it in 1.2 for estimators that had normalize=False by default and to change the default to False, to be then removed in 1.4, for estimators that had normalize=True by default.

Aha! Now a few more things make sense :D